test/acceptance/helpers/index.js

```diff
@@ -138,6 +139,15 @@ const startGravity = (port) =>
         expires_in: moment().add(100, 'days').utc().format()
       })
     })
+    app.get('/api/v1/profile/pace-gallery', (req, res) => {
+      res.send(fabricate('profile', {
+        owner_type: 'PartnerGallery',
+        owner: fabricate('partner', {
+          id: 'pace-gallery',
+          profile_layout: 'gallery_one'
+        })
+      }))
+    })
```

Comment:
Maybe this shouldn't respond to the specific `pace-gallery` profile route but a more generic partner profile? Or just `/api/v1/profile/*`. That way it can be re-used (+ it already is returning a fabrication).